server/controllers/refer/referralsController.test.ts

```diff
@@ -175,27 +177,46 @@ describe('ReferralsController', () => {
 
   describe('checkAnswers', () => {
     it('renders the referral check answers page', async () => {
+      const courseParticipations = [
+        courseParticipationFactory.build({ courseId: 'an-ID', createdAt: '2023-01-01T12:00:00.000Z' }),
+        courseParticipationFactory.build({
+          courseId: undefined,
+          createdAt: '2022-01-01T12:00:00.000Z',
+          otherCourseName: 'Another course',
+        }),
+      ]
+      const organisation = organisationFactory.build({ id: courseOffering.organisationId })
       const person = personFactory.build()
-      personService.getPerson.mockResolvedValue(person)
```

Comment:
I think easier in terms of test length (less scrolling, and maybe a little easier to spot the start of a test when there are fewer small groups) and grouping statements by the kind of thing that's being done rather than the kind of entity it's being done to, but I can see the benefit in keeping creation and mocking together. Don't have a strong preference - happy to switch back!